### ISSUE TITLE
Revert Redirect in Education: VET TEC Training Providers

### ISSUE THREAD

**Issue by lunascoop** [department-of-veterans-affairs/va.gov-team#7853]
### Description
This page was redirected on 4/06 but is causing a looping error.

**This is an updated ticket based on work completed in ticket #2052 ** 

### Requestor team info
Public Websites

### Implementation date
**ASAP****

### Redirects needed

Redirect back to URL  |  Current URL
---  |  ---
www.benefits.va.gov/GIBILL/FGIB/VetTecTrainingProviders.asp | /education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/ this isn't a Veteran facing page

### Definition of done
- [x] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
- [x] All appropriate VA stakeholders are notified of pending redirect - *@ requesting team*
- [x] Request is assigned to appropriate team for implementation - *@ Content & IA team*
- [x] Implementation team completes work - *@ Implementation team*
- [x] Implementation pushed live and redirect is validated in production - *@ all*
- [x] Ticket is closed - *@ requesting team*

**Comment by AnneHurley** [comment]
This was in original ticket:
![image](https://user-images.githubusercontent.com/46570591/78998276-1ff9ca80-7afd-11ea-92d2-906f704d38da.png)

**Comment by lunascoop** [comment]
@AnneHurley I'm also not sure what "this isn't a Veteran facing page" means. I did review the previous ticket re: VET TEC providers that @Mikki included in the [Slack convo](https://dsva.slack.com/archives/C52CL1PKQ/p1586530095328000?thread_ts=1586527307.312100&cid=C52CL1PKQ). From what I gathered in ticket [#18427](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18427), the VET TEC providers legacy content was supposed to be included on the current VET TEC page, perhaps in accordion style but was not. Therefore, the redirect VET TEC providers legacy page would be implemented since the content would be migrated to the main VET TEC page.

Perhaps the VET TEC content needs to be updated to include the VET TEC providers list. @peggygannon can you provide some insight?

**Comment by lunascoop** [comment]
**_More info per Jen's slack comment:_** "That page initially HAD been migrated into the new va.gov page. It was one of several things that caused the firedrill last winter/this spring. The stakeholders informed us that that contacts list changes all the time, sometimes weekly or daily, they are updating something on that list. As part of the immediate content updates we had to make on that page, we removed the contacts list and put back the link pointing to their up-to-date contacts info."
